examples/chart/teleport/scripts/install-on-minikube

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+base_dir=$(cd $(dirname $0); cd ..; pwd)
+
+cd $base_dir
+
+scripts/create-internal-pki
+
+cd $base_dir/pki
+
+pushd ..
+kubectl create secret generic license --from-file=license-enterprise.pem
+popd
+kubectl create secret tls tls-web --cert=proxy-server.pem --key=proxy-server-key.pem
+kubectl create configmap ca-certs --from-file=ca.pem
+
+cd $base_dir
+
+helm upgrade --install --reset-values -f minikube.values.yaml --recreate-pods --wait teleport ./
+
+pod=$(kubectl get po | grep teleport | grep Running | cut -d' ' -f1)
+
+kubectl exec -it $pod -- bash -c "tctl users add --roles=admin $USER" | tee tctl.log
+
+kubectl port-forward $pod 3022 3023 3024 3025 3026 3080 1>&2 > port-forward.log &
```

Comment:
remove extra port forwards now that we know they are not necessary

Reply:
At least 3080 and 3026 are required in order to test it locally via `teleport.example.com` resolved via the local resolver config. 

> remove extra port forwards

So, maybe you are talking about removing 3022, 3023, 3024 and 3025 specifically, right?